Update take_action.md (#36562)
DataDog/documentation: content/en/bits_ai/bits_ai_sre/take_action.md

- /bits_ai/bits_ai_sre/remediate_issues/
---

After Bits AI SRE identifies a root cause, it can help you take action in three ways:

- Code Fixes help resolve code-related root causes.

> - Remediation Recommendations provide corrective steps and fully configured commands.

Reviewer: This sentence chains three verbs ("connects," "iterates," "uses") and runs to 30 words. Consider splitting:

"The Dev Agent connects to GitHub or GitLab to create production-ready pull requests. It iterates on fixes using CI logs and developer feedback and uses multiple Datadog products to generate contextual fixes."

- Triage Suggestions help coordinate the response through connected tools.

> 

Reviewer: Minor: "has determined" could be present tense for consistency with the rest of the page.

```suggestion
To start receiving code fixes, [set up the Bits AI Dev Agent][1]. Then, if Bits AI SRE determines a code-related root cause, you automatically receive suggested code fixes.
```

> ## Code Fixes

Reviewer: ```suggestion
## Code fixes
```

> Bits AI SRE integrates with [Bits AI Dev Agent][2] to automatically generate code fixes. The Dev Agent connects to GitHub or GitLab to create production-ready pull requests and iterates on fixes using CI logs and developer feedback, and uses multiple Datadog products to generate contextual fixes.

Reviewer 2: Parallelism is broken here — `to **create**…and **iterates**…and **uses**` mixes infinitive and present-tense verbs, and the double `and` makes the sentence run on.

```suggestion
Bits AI SRE integrates with [Bits AI Dev Agent][2] to automatically generate code fixes. The Dev Agent connects to GitHub or GitLab to create production-ready pull requests, iterate on fixes using CI logs and developer feedback, and apply context from multiple Datadog products.
```

> To start receiving code fixes, [set up the Bits AI Dev Agent][1]. Then, if Bits AI SRE has determined a code-related root cause, you will automatically receive suggested code fixes.

Reviewer 2: Per [datadog-vale](https://github.com/DataDog/datadog-vale), avoid temporal words like `will`.

```suggestion
To start receiving code fixes, [set up the Bits AI Dev Agent][1]. Then, if Bits AI SRE determines a code-related root cause, you receive suggested code fixes automatically.
```

> To start receiving code fixes, [set up the Bits AI Dev Agent][1]. Then, if Bits AI SRE has determined a code-related root cause, you will automatically receive suggested code fixes.

Reviewer: Avoid temporal "will" — docs should be evergreen. Use present tense.

```suggestion
Bits AI SRE recommends corrective steps when applicable for the root cause. These recommendations can include commands that are fully configured, so you can copy them and run in your CLI. For example, Bits AI SRE may provide a `kubectl` command to restart or patch an affected Kubernetes deployment.
```

{{< img src="bits_ai/bits_ai_sre_suggested_code_fix.png" alt="Flowchart showing Bits' investigation conclusion and a suggested code fix" style="width:100%;" >}}

> ## Remediation Recommendations

Reviewer: Missing blank line between the `## Triage Suggestions` heading and the paragraph. The other headings on this page have a blank line — keep it consistent.

> 

Reviewer: "Bits" should be "Bits AI SRE" for consistency with the rest of the page. Also, "take tasks" isn't idiomatic — "complete tasks" reads better.

```suggestion
Triage Suggestions help you coordinate the response without leaving the investigation page. Bits AI SRE uses investigation context to prefill messages, incident descriptions, or ticket metadata, so you can complete common response tasks with a single click.
```

> Bits AI SRE will recommend corrective steps when applicable for the root cause. These recommendations can include commands that are fully configured, so you can copy them and run in your CLI. For example, Bits AI SRE may provide a `kubectl` command to restart or patch an affected Kubernetes deployment.

Reviewer 2: A few issues stacked on this line:
- `will` violates the Vale rule against temporal words
- `applicable **for**` should be `applicable **to**`
- `run in your CLI` is missing an object
- The remediation feature also supports AWS, GCP, Azure, Terraform, and database categories — leading with only `kubectl` makes it sound K8s-specific

```suggestion
Bits AI SRE recommends corrective steps when applicable to the root cause. These recommendations can include fully configured commands that you copy and run in your terminal — for example, a `kubectl` command to restart a Kubernetes deployment, an AWS CLI call to adjust a resource, or a Terraform change to update infrastructure.
```

Review each recommendation and command before running it in your environment.

## Triage Suggestions

> Triage Suggestions help you coordinate the response without leaving the investigation page. Bits uses investigation context to prefill messages, incident descriptions, or ticket metadata, so you can take common response tasks with a single click.

Reviewer 2: Three small things:
1. Missing blank line after the H2 (the other two sections have one).
2. `Bits uses…` — every other reference in this page says `Bits AI SRE`. Worth being consistent.
3. `take common response tasks` reads oddly — tasks are completed or performed, not taken.

```suggestion
## Triage Suggestions

Triage Suggestions help you coordinate the response without leaving the investigation page. Bits AI SRE uses investigation context to prefill messages, incident descriptions, or ticket metadata, so you can complete common response tasks with a single click.
```

> Suggestions include but are not limited to

Reviewer 2: Missing colon before the bulleted list.

```suggestion
Suggestions include but are not limited to:
```

> - Sending Slack messages to update your team on the investigation results

Reviewer 2: The previous version of this list also included Microsoft Teams messages, PagerDuty incidents, and Datadog Case Management cases. Was dropping those intentional? If they're still supported, the shorter list undersells what triage suggestions can do.

- Declaring incidents in Datadog

> - Paging engineers via Datadog On-Call

Reviewer 2: Per the Vale rules, prefer `through` or `with` over `via`.

```suggestion
- Paging engineers through Datadog On-Call
```
